client/src/interface.rs

```diff
@@ -98,16 +148,29 @@ impl Stronghold {
     }
 
     /// Switches the actor target to another actor in the system specified by the client_path: `Vec<u8>`.
-    pub fn switch_actor_target(&mut self, client_path: Vec<u8>) -> StatusMessage {
+    pub async fn switch_actor_target(&mut self, client_path: Vec<u8>) -> StatusMessage {
         let client_id = ClientId::load_from_path(&client_path, &client_path.clone()).expect(line_error!());
 
         if self.client_ids.contains(&client_id) {
             let idx = self.client_ids.iter().position(|cid| cid == &client_id);
 
             if let Some(idx) = idx {
                 self.current_target = idx;
-            }
 
+                #[cfg(feature = "communication")]
+                match ask(
```

Comment:
does this mean that every `switch_actor_target` call changes both the local stronghold client and the connected strongholds?

Reply:
It changes the client that receives incoming request to be the one that is now also targeted by the local stronghold. I could implement this in a separate method, but imo it makes sense that if the user switches the client, the remote request should be forwarded to that new client. But maybe we have to discuss this again for other use cases

Reply:
This should be fine as is. We can work with other use-cases when as they come.